Packs/CVESearch/TestPlaybooks/playbook-CVE-Search-Test.yml

```diff
@@ -1,10 +1,10 @@
 id: CVE Search v2 - Test
 version: -1
 name: CVE Search v2 - Test
```

Comment:
please make sure this test still runs and passes as expected

Reply:
Did that, It was my bad to make the id's uuids to begin with

Reply:
so it passes?

Reply:
Yup